tech/languages/rust/further-reading.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Further reading
+subsection: rust
+order: 4
+---
+
+# Learning resources
+
+There is an official book describing the Rust programming language in detail.
+https://doc.rust-lang.org/book/2018-edition/index.html
+
+More examples of the concepts discussed in the book are available in this collection:
+https://doc.rust-lang.org/rust-by-example/
+
+When you finish with the language itself, there is an ongoing effort to create a set of examples of using core libraries for concurrency, data serialization, network programming, etc.
+https://rust-lang-nursery.github.io/rust-cookbook/intro.html
+
+Finally, you can find community pages with a list of resources such as this one:
+https://github.com/ctjhoa/rust-learning
+
+
+# API documentation
+
+Standard library documentation:
+https://doc.rust-lang.org/std/
+
+Documentation for crates available at `crates.io`:
+https://docs.rs/
+
+# IDE recommendations
+
+There are plugins for popular editor such as Vim, Atom or Gnome Software which are based on the Rust Language Server (RLS) implementation. Alternatively, IntelliJ IDEA provides their own, custom plugin. Unfortunatelly, the debugging support is available only in paid CLion IDE.
+https://areweideyet.com/
```

Comment:
It'd be best if you can make some recommendation / guide from this. Ask yourself a question "how can user do this on Fedora?"

Reply:
(s)he can install atom/vscode/intellij ... none of them is packaged for Fedora. Maybe it would be more useful to write a section about the installation of these editors/ides. The only editor available in Fedora is Vim (maybe Emacs, if anyone is really using it), which can use the language server implementation with some plugins, but I don't want to write a tutorial on that, because it will break with the next upstream release of vim, vundle, rls or whatever.

Reply:
Well, that's important information and could be standalone section indeed. 
No extensive "tutorial" is needed (or wanted) here. What we need is just basic info to send a developer in the right direction (for him to find and get the right "thing" he/she needs and some sources/references for further inqueries).